dynamodb/src/main/scala/uk/gov/nationalarchives/DADynamoDBClient.scala

```diff
@@ -23,32 +25,88 @@ import scala.language.postfixOps
   */
 class DADynamoDBClient[F[_]: Async](dynamoDBClient: DynamoDbAsyncClient) {
 
-  /** Gets an attribute from dynamoDB
-    * @param dynamoDbRequest
-    *   a case class with table name, primary key and value as well as the name of the attribute
+  /** Writes the list of items of type T to Dynamo
+    * @param tableName
+    *   The name of the table
+    * @param items
+    *   A list of items to write to Dynamo
+    * @param format
+    *   An implicit scanamo formatter for type T
+    * @tparam T
+    *   The type of the items to be written to Dynamo
     * @return
-    *   The response from the get attribute call wrapped with F[_]
+    *   The BatchWriteItemResponse wrapped with F[_]
     */
-  def getAttributeValues(dynamoDbRequest: DynamoDbRequest): F[Map[String, AttributeValue]] = {
-    val getAttributeValueRequest = GetItemRequest
-      .builder()
-      .tableName(dynamoDbRequest.tableName)
-      .key(dynamoDbRequest.primaryKeyAndItsValue asJava)
-      .build()
-
+  def writeItems[T <: Product](tableName: String, items: List[T])(implicit
```

Comment:
I was thinking, should we add "batch" to be the beginning of it to differentiate it from the non-batch method(s)?

Reply:
Same comment as above.